<launch>
	<node pkg="dynamic_parameter" type="arm_param_server.py" name="arm_param_server">
	</node> 
	<node pkg="dynamic_parameter" type="gripper_param_server.py" name="gripper_param_server">
	</node> 

	<node pkg="dynamic_parameter" type="flag_param_server.py" name="flag_param_server">
	</node> 

	<node name="$(anon dynparam)" pkg="dynamic_reconfigure" type="dynparam" args="set_from_parameters hokuyo_node">
    <param name="min_ang" type="double" value="-1.0" />
    <param name="max_ang" type="double" value="1.0" />
  </node>
	
</launch>